File: ArrayAlgos/Kadanes algo/sol1.py
def kadanesAlgorithm(array):
    # Write your code here.
        csum = array[0]
        osum = array[0]
	
        for i in range(1,len(array)):
            if csum >=0:
                csum+=array[i]
            else:
                csum = array[i]
            if csum > osum:
                osum = csum
        return osum	

File: Sorting/Bubble Sort/sol1.cpp
#include <vector>
#include <algorithm>
using namespace std;

vector<int> bubbleSort(vector<int> array) {
  // Write your code here.
  for(int i=1; i < array.size(); i++){
      for(int j=0;j < array.size()-i;j++){
          if(array[j]>array[j+1]){
						swap(array[j], array[j+1]);
					}
      }
  }
  return array;
}
